docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -473,20 +473,20 @@ Refer to the [secrets-config proxy](../../security/secrets-config-proxy/) docume
 !!! example
     Given the following files created outside the scope of this document:
     
-    * `nginx.crt` user-provided certificate (replacing the default)
-    * `nginx.key` user-provided private key (replacing the default)
+    * `server.crt` user-provided certificate (replacing the default)
+    * `server.key` user-provided private key (replacing the default)
     * `ca.pem` certificate authority file (that signed `nginx.crt`, directly or indirectly)
```

Comment:
If you're going to rename to server.crt and server.key, you should also rename nginx.crt in the 'ca.pem' line as well.